dnplab/__init__.py

```diff
@@ -1,6 +1,7 @@
 # import odnpData
 from .dnpData import dnpdata, dnpdata_collection, create_workspace
 from . import dnpNMR
+from . import dnpImport
```

Comment:
Can you please make this change?
```suggestion
from .dnpImport import load
```